app/managers.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("cinema")
+        self.table = "actors"
+
+    @staticmethod
+    def check_name(first_name: str, last_name: str) -> list:
+        if isinstance(first_name and last_name, str):
```

Comment:
This check will not work correctly, separate this check between two conditions. E.g., `isinstance(3 and "a", str)` will return `True`.

Reply:
this is so sad, I thought it would work